pandas/core/missing.py

```diff
@@ -447,7 +452,6 @@ def wrapper(arr, mask, limit=None):
 
 
 def pad_1d(values, limit=None, mask=None, dtype=None):
-
```

Comment:
Ok... Sorry for that... I'm using IntelliJ IDEA, and it formatted all file with PEP8 standard